index.js

```diff
@@ -106,20 +138,50 @@ function testGenResponse(swagger, path, operation, response, config) {
  * @param  {string} path API path to generate tests for
  * @param  {string} operation operation of the path to generate tests for
  * @param  {json} config configuration for testGen
+ * @param  {json} parameters json for different parameters
  * @returns {string|Array} set of all tests for a path's operation
  */
-function testGenOperation(swagger, path, operation, config) {
+function testGenOperation(swagger, path, operation, config, parameters) {
   var responses = swagger.paths[path][operation].responses;
   var result = [];
   var res;
+  var param;
+  var parameterType = swagger.paths[path][operation].parameters;
+  var type;
 
+  // body parameters to payload
+  if (swagger.paths[path][operation].hasOwnProperty('parameters')) {
+    // process different parameters
+    for (param in parameterType) {
+      if (parameterType.hasOwnProperty(param)) {
+        type = parameterType[param];
+        switch (type.in) {
```

Comment:
`fromData` case is missing. See [the sepc](https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#parameterObject)